tests/unit/Application/Metadata/OauthClientCredentialsClientJsonGeneratorTest.php

```diff
@@ -64,14 +64,16 @@ public function test_it_builds_an_entity_change_request()
         $diff = $entity->diff($changedEntity);
         $contact = m::mock(Contact::class);
         $contact->shouldReceive('getEmailAddress')->andReturn('j.doe@example.com');
-        $data = $generator->generateEntityChangeRequest($entity, $diff, $contact);
+        $contact->shouldReceive('getDisplayName')->andReturn('A.F.Th. van der Heijden');
+        $data = $generator->generateEntityChangeRequest($entity, $diff, $contact, 'CHR-421');
 
         $this->assertIsArray($data);
         $this->assertEquals('manageId', $data['metaDataId']);
         $this->assertEquals('oidc10_rp', $data['type']);
         $this->assertIsArray($data['pathUpdates']);
         $this->assertCount(2, $data['pathUpdates']);
-        $this->assertSame('revisionnote', $data['note']);
+        $this->assertStringContainsString('Change request by user A.F.Th. van der Heijden with email address "j.doe@example.com" via the SPdashboard', $data['note']);
```

Comment:
Just for thoughts... shouls we use use a variable for the users name here, to ensure consistency in checking?

Reply:
Nah, this is just a test